MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -35,6 +35,12 @@
 #import "MSALStressTestHelper.h"
 #import "MSALPublicClientApplication+Internal.h"
 #import "MSIDDefaultTokenCacheAccessor.h"
+#import <WebKit/WebKit.h>
+
+#define TEST_EMBEDDED_WVTYPE @"Embedded"
```

Comment:
nit: can we spell webviewtype out please :)

Reply:
Sure, done